Array/ArrayHomework2.php

```diff
@@ -0,0 +1,37 @@
+<?php
+$calculate = array(
+    array(5, 12, 17, 9, 3),
+    array(13, 4, 8, 14, 1),
+    array(9, 5, 3, 7, 21)
+);
+
+
+
+for ($i = 0; $i < 3; $i++) {
+    $avg = 0;
+    $sum = 0;
+    $min = $calculate[0][0];
+    $max = $calculate[0][0];
+    for ($j = 0; $j < 5; $j++) {
```

Comment:
`$j < 5;` you counted number of each array in your mind and put here, you shouldn't do it. Let computer count number of elements by `count` or `sizeof`. Please fix for above also.